manim/mobject/text/text_mobject.py

```diff
@@ -432,13 +433,16 @@ def __init__(
         width: float = None,
         should_center: bool = True,
         disable_ligatures: bool = False,
+        use_svg_cache: bool = False,
         **kwargs,
     ) -> None:
         self.line_spacing = line_spacing
         if font and warn_missing_font:
-            fonts_list = manimpango.list_fonts()
-            if font not in fonts_list:
-                logger.warning(f"Font {font} not in {fonts_list}.")
+            Text.font_list = (
+                Text.font_list if len(Text.font_list) > 0 else manimpango.list_fonts()
+            )
```

Comment:
More of a semantical question: is there any particular argument against implementing this as a cached method of `Text`? Like

```py
class Text(...):
    ...
    
    @staticmethod
    @functools.lru_cache(maxsize=None)
    def font_list() -> list[str]:
        return manimpango.list_fonts()
```

Reply:
I can bench it later on, good point but might make it still more inefficient because it checks the cache but it is also in c, so who knows what will happen.

I give report as soon as i benched it